EMS/core-bundle/src/Routes.php

```diff
@@ -74,6 +74,10 @@ class Routes
     final public const string FILTER_EXPORT = 'emsco_filter_export';
     final public const string FORM_ADMIN_INDEX = 'emsco_form_admin_index';
     final public const string FORM_ADMIN_ADD = 'emsco_form_admin_add';
+    final public const string GROUP_ADD = 'emsco_group_admin_add';
+    final public const string GROUP_EDIT = 'emsco_group_admin_edit';
+    final public const string GROUP_DELETE = 'emsco_group_admin_delete';
+    final public const string DELETE_ALL_GROUP = 'emsco_group_admin_delete_all_group';
```

Comment:
Bad naming DELETE_ALL_GROUP -> GROUP_DELETE_ALL

Reply:
We don't need a delete all group button